src/CoreShop2VueStorefrontBundle/Bridge/ElasticsearchImporter.php

```diff
@@ -64,9 +64,9 @@ public function import(callable $callback): void
         }
 
         foreach ($listing as $object) {
-            $this->persister->persist($object);
-
             $callback($object);
+
+            $this->persister->persist($object);
```

Comment:
Moving the callback before the object persister allows it to output info about the object. In the future it could also do other things to it, maybe even modifying it (but would need to return it)